common/STcommon.py

```diff
@@ -67,7 +67,7 @@ def parseArgs():
 	'''Parses args using the argparse lib'''
 	parser = argparse.ArgumentParser(description='Location logging server')
 
-	parser.add_argument('-c', '--configure', nargs=2, metavar='ADDRESS PORT')
+	parser.add_argument('-c', '--configure', nargs=3, metavar='ADDRESS PORT LOGFILE')
```

Comment:
Should we consider using the -c as a flag to write to file or 'change', then use like -a, -p, -l as flags to set each param individually. thoughts?